### DISCUSSION TITLE
Code block

### DISCUSSION THREAD

**Discussion by pushself** [athensresearch/athens#1211] [Feature Requests]
Hope to have a code block function similar to roam research.
The current triple quotation marks are too difficult to use. Pasting from the code editor will result in multiple new blocks

![image](https://user-images.githubusercontent.com/30368301/119210530-34214100-badf-11eb-9e86-3048e9107d4f.png)

**Comment by alexandergunnarson** [comment]
One note —

> Pasting from the code editor will result in multiple new blocks

Jeff Tang helped me to realize that you can just cmd+alt+shift+V to paste in just one block.
